packages/core/src/mpi/match-patients.ts

```diff
@@ -297,6 +308,27 @@ export function epicMatchingAlgorithm(
   return match;
 }
 
+export function strictMatchingAlgorithm(patient1: PatientData, patient2: PatientData): boolean {
```

Comment:
This should also get unit tests 🙏

Reply:
At the bottom of the Unit test file there should be test specifc to testing that out albeit through the validateCwLinksBelongToPatient. Let me know if thats fine or we want more granular tests